deploy/helm/metrics-collector-config.yaml

```diff
@@ -1320,6 +1292,22 @@ receivers:
               - "container_network_transmit_packets_total"
               - "container_network_receive_packets_dropped_total"
               - "container_network_transmit_packets_dropped_total"
+              - "apiserver_request_total"
+              - '{__name__="kubernetes_build_info", job=~".*apiserver.*"}'
+{{- if .Values.otel.metrics.extra_scrape_metrics }}
+{{ toYaml .Values.otel.metrics.extra_scrape_metrics | indent 14 }}
```

Comment:
Are we going to check, that in extra_scrape_metrics isn't defined KSM metric?

Reply:
intended use of extra_scrape_metrics are metrics from prometheus that customer needs (not us), so this should be convering only prometheus scraping and I assume that if use would need some addition kube_ metrics he would have them in prometheus and add them here are normally